master/buildbot/worker_transition.py

```diff
@@ -199,7 +201,9 @@ def deprecatedWorkerModuleAttribute(scope, attribute, compat_name=None,
         "scope must be module, i.e. locals()"
 
     if new_name is None:
-        attribute_name = scope.keys()[scope.values().index(attribute)]
+        scope_keys = list(iterkeys(scope))
```

Comment:
@tardyp do you want me to get rid of itervalues() also, and use list(scope.values()) ?